Core/Core/DocViewer/PSPDFAnnotationExtension.swift

```diff
@@ -58,12 +58,11 @@ extension PSPDFAnnotation {
             annotation = strikeout
         case .freetext:
             let freeText = PSPDFFreeTextAnnotation(contents: apiAnnotation.contents ?? "")
-            let fontInfo = apiAnnotation.font?.split(separator: " ")
             let fontSizeStr = apiAnnotation.font?.trimmingCharacters(in: CharacterSet(charactersIn: "0123456789").inverted) ?? ""
-            freeText.fontName = fontInfo?.last.flatMap { String($0) } ?? "Verdana"
-            freeText.fontSize = CGFloat(Int(fontSizeStr) ?? 14)
-            freeText.fillColor = .white
-            freeText.sizeToFit()
+            freeText.fontName = "Helvetica" // apiAnnotation.font?.split(separator: " ")?.last.flatMap { String($0) } ?? "Helvetica"
```

Comment:
Did you mean to leave the comment?

Reply:
yes. I haven't heard back from DocViewer peeps if always using Helvetica is intentional. This logic isn't anywhere else and I didn't want to lose it.